docs/sources/developers/http_api/user.md

```diff
@@ -24,11 +24,15 @@ refs:
 
 # User API
 
-The Users HTTP API does not currently work with an API Token. API Tokens are linked to an organization and an organization role. They cannot be given
-the permission of server users access, only users can be given that permission. To use these API calls you can use Basic Auth and the Grafana
-user must have the Grafana Admin role.
+{{% admonition type="caution" %}}
+The Users HTTP API currently does not work with service account tokens. Service accounts are limited to an organization and an organization role. They cannot be granted [Grafana server administrator permissions]({{< relref "../roles-and-permissions/#grafana-server-administrators" >}}).
+In order to use these API endpoints you have to use Basic Auth and the Grafana user must have the Grafana server administrator permission.
 
-API Tokens can be used with Organization HTTP API to get users of specific organization.
+The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
+
+Service account tokens can be used with Organization HTTP API to get users of specific organization.
+
+{{% /admonition %}}
```

Comment:
- Removes "currently" to make the documentation more timeless: https://developers.google.com/style/timeless-documentation
- Updates the shortcode syntax to match our preferred form which has more consistent semantics (I will open a PR to rewrite all the existing ones later): https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanashortcodes
- Replaces the broken link that uses the `relref` shortcode with a working link that follows our linking guidance: https://grafana.com/docs/writers-toolkit/write/links/
- Modified some language for readability and tone

```suggestion
{{< admonition type="caution" >}}
You can't authenticate to the User HTTP API with service account tokens.
Service accounts are limited to an organization and an organization role.
They can't be granted [Grafana server administrator permissions](/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/#grafana-server-administrators).

To use these API endpoints you have to use Basic authentication and the Grafana user must have the Grafana server administrator permission.

The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
{{< /admonition >}}
```